tests/test_docs.py

```diff
@@ -2,6 +2,7 @@
 import inspect
 
 import sqlglot
+import sqlglot.optimizer
 
```

Comment:
why is this needed? we’re the test broken?

Reply:
Yeah, it wasn't running locally. Might be a setup quirk on my side... but this felt harmless.